robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowValueAnimator.java

```diff
@@ -2,13 +2,22 @@
 
 import android.animation.ValueAnimator;
 
+import org.robolectric.annotation.Implementation;
 import org.robolectric.annotation.Implements;
+import org.robolectric.annotation.RealObject;
 import org.robolectric.annotation.Resetter;
 import org.robolectric.util.ReflectionHelpers;
 
+import static org.robolectric.internal.Shadow.directlyOn;
+
 @Implements(ValueAnimator.class)
 public class ShadowValueAnimator {
 
+  @RealObject
+  private ValueAnimator m_realObject;
```

Comment:
Please don't use "m_" prefixes for fields. It's not a convention we've adopted for the codebase.